src/index.js

```diff
@@ -3,5 +3,11 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import 'bootstrap/dist/css/bootstrap.css';
+import { BrowserRouter as Router } from 'react-router-dom';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <Router>
+    <App />
+  </Router>,
+  document.getElementById('root')
+);
```

Comment:
Good work. You'll see some devs wrap their `<App />` with `<Router />` in their App.js, but I prefer seeing it in index like Dustin has been doing. Looks how it should look.